flow360/component/flow360_params/flow360_params.py

```diff
@@ -491,10 +491,16 @@ class Geometry(Flow360BaseModel):
     :class: Geometry component
     """
 
-    ref_area: Optional[AreaType.Positive] = pd.Field(alias="refArea", displayed="Reference area")
-    moment_center: Optional[LengthType.Point] = pd.Field(alias="momentCenter")
-    moment_length: Optional[LengthType.Moment] = pd.Field(alias="momentLength")
-    mesh_unit: Optional[LengthType] = pd.Field(alias="meshUnit")
+    ref_area: Optional[AreaType.Positive] = pd.Field(
+        alias="refArea", displayed="Reference area", default_factory=lambda: 1
+    )
+    moment_center: Optional[LengthType.Point] = pd.Field(
+        alias="momentCenter", default_factory=lambda: (0.0, 0.0, 0.0)
+    )
+    moment_length: Optional[LengthType.Moment] = pd.Field(
+        alias="momentLength", default_factory=lambda: (1.0, 1.0, 1.0)
+    )
+    mesh_unit: Optional[LengthType] = pd.Field(alias="meshUnit", default_factory=lambda: 1)
```

Comment:
In pydantic V1 default_factory causes alot of issues, most importantly this one https://github.com/pydantic/pydantic/issues/1717 which breaks the updater which uses parse_obj extensively. There are other more subtle issues with default_factories on optional fields like confusing error messages.

@maciej-flexcompute maybe we should consider migrating to pydantic v2